docs/boot-options.rst

```diff
@@ -215,10 +215,16 @@ inst.ks
 Give the location of a kickstart file to be used to automate the install.
 Locations may be specified using any of the formats allowed for `inst.repo`_.
 
-For any format the ``<path>`` component defaults to ``/ks.cfg`` if it is omitted.
 
-For NFS kickstarts, if the ``<path>`` ends in ``/``, ``<ip>-kickstart`` is added.
+``inst.ks=file://<path>/<filename>``
+    For any format the ``<path>`` component defaults to ``/ks.cfg`` if it is omitted.
+    
+    NOTE: The path must be absolute and start with `/` so the file url starts
+    with `file:///...`.
 
+``inst.ks=nfs:<next_server>:<path>/<filename>````
+    For NFS kickstarts, if the ``<path>`` ends in ``/``, ``<ip>-kickstart`` is added.
+ 
```

Comment:
- This suffers from the same problem with `/<filename>` being redundant.
- `next_server` - just `server` please, boot options are manually entered, not from DHCP
- Perhaps this should be also introduced somehow?